### DISCUSSION TITLE
About add product

### DISCUSSION THREAD

**Discussion by Andy487** [shopizer-ecommerce/shopizer#839] [Q&A]
I am trying to add product but getting error... please suggest 

![image](https://user-images.githubusercontent.com/54777475/201023209-3200ff0c-0a6a-4889-bc05-5510a66bdac2.png)

![](https://files.slack.com/files-pri/TCQKQPA9W-F04AADTFA2Y/image.png)

**Comment by chekotisrikanth** [comment]
seems even for v2, url for product creation and get are same as v1. 
so in shopizer-admin-main\src\app\pages\catalogue\products\services\product.service.ts
remove /definition from all places (2 places) save file. server will restart if not restart manually.

**Reply by RoxerGit** [reply]
i have the same issue,  thanks for the solution :), my concern is that backend and frontend are not in sync

**Reply by jaffarali0272** [reply]
I think your are using wrong endpoint 
api/v2/private/product try with this one.
